lib/internal/modules/cjs/loader.js

```diff
@@ -597,14 +600,28 @@ Module._resolveLookupPaths = function(request, parent, newReturn) {
 //    Then have it load  the file contents before returning its exports
 //    object.
 Module._load = function(request, parent, isMain) {
+  let identifier;
```

Comment:
Perhaps call this something more descriptive like `relResolveCacheIdentifier`?

Reply:
Updated.